.github/workflows/pr_gate.yml

```diff
@@ -4,6 +4,11 @@ on:
   pull_request:
     branches:
     - main
+    paths:
+    - '**'
+    - '!README.md'
+    - '!LICENSE'
+    - '!database/**'
```

Comment:
```
    paths:
    - '**'
    - '!README.md'
    - '!LICENSE'
    - '!database/**'
```
It takes everything and it doesn't ignore readme

```
    paths:
    - '!README.md'
    - '!LICENSE'
    - '!database/**'
```
It ignores everything

Reply:
```
    paths-ignore:
    - README.md
    - LICENSE
    - 'database/**'
```
it doesn´t ignore a change in README, see https://github.com/miguelgfierro/pybase/runs/7486032459?check_suite_focus=true

Reply:
```
    paths-ignore:
    - 'README.md'
    - 'LICENSE'
    - 'database/**'
    - '.github/**'
    - '.gitignore'
```
it doesn't work